docs/manual/macros.md

```diff
@@ -71,57 +71,57 @@ to perform useful operations. The current list is
 	%getconfdir	expand to rpm "home" directory (typically /usr/lib/rpm)
 	%dnl		discard to next line (without expanding)
 	%verbose	expand to 1 if rpm is in verbose mode, 0 if not
-	%{verbose:...}	expand to ... if rpm is in verbose mode, the
+	%{verbose:…}	expand to … if rpm is in verbose mode, the
```

Comment:
I don't think the use of unicode ellipsis ("…") is an improvement at all. It turns a plain and simple thing into something you have to look up how to type, and readability becomes very renderer-dependent. In this comment box, it's rendered reasonably, but in the above line and my terminal the three dots are so close together it's just unreadable with tired eyes.

Reply:
Fair enough! I'll drop that change from the PR.